packages/quip-apps-api/src/client.ts

```diff
@@ -66,6 +66,8 @@ export interface ToolbarState {
     toolbarCommandIds?: string[];
     disabledCommandIds?: string[];
     highlightedCommandIds?: string[];
+    destructiveCommandIds?: string[];
+    mobileCommandIds?: string[];
```

Comment:
Actually thinking about it now, what do you think about renaming mobileCommandIds -> mobileToolbarCommandIds? This will make it more explicit when third party developers want to use this.

Reply:
sounds good